APMrover2/Log.cpp

```diff
@@ -510,9 +510,9 @@ void Rover::Log_Write_Vehicle_Startup_Messages()
 // start a new log
 void Rover::start_logging()
 {
-    in_mavlink_delay = true;
+    DataFlash.EnableWrites(false);
     DataFlash.StartUnstartedLogging();
-    in_mavlink_delay = false;
+    DataFlash.EnableWrites(true);
```

Comment:
I know this is almost a NFC, but I wonder why it is needed here - I don't see it in any other vehicle.